responsive-css.md

```diff
@@ -0,0 +1,123 @@
+# Styling your website to be responsive
+
+## Viewport
+
+The viewport is the part of the webpage that's currently visible to the user. Viewports vary between different devices,
+as the screen size between devices also varies. For example, the viewport of a desktop monitor will be significantly larger than the viewport of
+a mobile device.
+
+The viewport can be controlled by setting a `<meta>` tag in each web page you'd like to make responsive. The following is an example of how to use this tag:
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+```
+
+`width=device-width` sets the width of the viewport to match the width of the device. This is useful for devices (such as mobile devices) that have less width compared to the height of the device.
+
+`initial-scale` sets the zoom factor of the viewport to the value of the scale provided. We set this to `1.0` to set it as the default zoom for the viewport when the webpage is first loaded.
+
+## Mobile-First Design
+
+Mobile-first design refers to the practice of designing your website for mobile devices before designing your website for desktop devices.
+
+A good methodology for this is to design your website to work on smaller screens, and then add additional styling for larger screen resolutions via `Media Queries`.
+
+## Media Queries
+
+A Media query is a way to add additional CSS styling based on the given condition of the query.
+
+The query follows the `@media` syntax, followed by the condition on which it should apply the additional CSS, followed by the additional CSS to be applied.
+
+The following example changes the background color of the body to black
+when the browser window's width is 600px or smaller.
```

Comment:
Might be useful to link to where this is done on the uwbhacks website

Reply:
Ehh, if I link to a specific line on the website, there's a possibility that it might change to something else. I was planning on adding a few CodeSandbox examples to this doc before merge, that way users can see this basic example in action.

Reply:
It won't change if you link to a line number of a specific commit